docs/setting-up-your-environment.md

```diff
@@ -97,13 +114,33 @@ To preview the website in your local browser, make sure you're in the
 `btcinformation.org` directory and run the following command:
 
     make preview
+    
+###### Mac differences:
+
+You might get an error that looks like this:
+```
+Conversion error: Jekyll::Converters::Scss encountered an error while converting 'css/main.scss':
+                    Invalid US-ASCII character "\xE2" on line 1917
+jekyll 3.0.1 | Error:  Invalid US-ASCII character "\xE2" on line 1917
+make: *** [preview] Error 1
+```
+
+For that, we will have to set some environment variables for the default language encoding 
+as per this thread: https://github.com/csswizardry/inuit.css/issues/270#issuecomment-56056606
+
+    export LC_ALL="en_US.UTF-8"
+    export LANG="en_US.UTF-8"
 
 This will compile the site (takes 5 to 10 minutes; see [the speed-up
 instructions](#fast-partial-previews-or-builds)) and then print a
 message like this:
 
     Server address: http://0.0.0.0:4000
     Server running... press ctrl-c to stop.
+    
+###### Mac differences:
+
+    Server address: http://127.0.0.1:4000//    
```

Comment:
Can you still access your server on MacOS at http://0.0.0.0:4000/ ?  (I think that should work even outside Linux, but I don't know for sure.)  If so, this difference might not need to be mentioned, but I'm fine with keeping it in if you think it's useful.

Reply:
I just tested it - yes, that 0.0.0.0:4000 address is accessible, but it wasn't apparent from the console output.